scripts/test/InelasticDirect_detpackmap_Test.py

```diff
@@ -0,0 +1,22 @@
+from Direct.detpackmap import sequoia
+import unittest
+
+
+class TestCase(unittest.TestCase):
+
+    def test_sequoia(self):
+        assert sequoia('B1') == 38
```

Comment:
Use `self.assertEquals` instead of plain `assert`. See for example [stackoverflow](https://stackoverflow.com/questions/6361147/why-do-assertions-in-unittest-use-testcase-assertequal-not-the-assert-keyword) for a discussion of differences between the two